lib/maze/server.rb

```diff
@@ -161,9 +162,9 @@ def start
             end
 
             # When adding more endpoints, be sure to update the 'I should receive no requests' step
-            server.mount '/notify', Servlets::Servlet, errors
-            server.mount '/sessions', Servlets::Servlet, sessions
-            server.mount '/builds', Servlets::Servlet, builds
+            server.mount '/notify', Servlets::Servlet, :errors
+            server.mount '/sessions', Servlets::Servlet, :sessions
+            server.mount '/builds', Servlets::Servlet, :builds
             server.mount '/uploads', Servlets::Servlet, uploads
             server.mount '/sourcemap', Servlets::Servlet, sourcemaps
             server.mount '/react-native-source-map', Servlets::Servlet, sourcemaps
```

Comment:
Don't these also need to use a symbol now?

```suggestion
            server.mount '/uploads', Servlets::Servlet, :uploads
            server.mount '/sourcemap', Servlets::Servlet, :sourcemaps
            server.mount '/react-native-source-map', Servlets::Servlet, :sourcemaps
```

Reply:
Absolutely, I must have got distracted mid way through.